tests/parsers/moldev_softmax_pro/to_allotrope_test.py

```diff
@@ -10,33 +10,26 @@
 
 
 @pytest.mark.parametrize(
-    "test_file,schema_file,expected_file",
+    "file_name",
     [
-        (
-            "tests/parsers/moldev_softmax_pro/testdata/fl_kinetic_plates.txt",
-            "fluorescence/BENCHLING/2023/09/fluorescence.json",
-            "tests/parsers/moldev_softmax_pro/testdata/expected_fl_kinetic.json",
-        ),
-        (
-            "tests/parsers/moldev_softmax_pro/testdata/lum_spectrum_columns.txt",
-            "luminescence/BENCHLING/2023/09/luminescence.json",
-            "tests/parsers/moldev_softmax_pro/testdata/expected_lum_spectrum.json",
-        ),
-        (
-            "tests/parsers/moldev_softmax_pro/testdata/abs_endpoint_plates.txt",
-            "ultraviolet-absorbance/BENCHLING/2023/09/ultraviolet-absorbance.json",
-            "tests/parsers/moldev_softmax_pro/testdata/expected_abs_endpoint.json",
```

Comment:
reiterating comment from deleted test files: can we keep the existing test data without deleting it and run tests as before? Or will they no longer pass?

Reply:
A change in the plate reader schema generates a change in the output, so all tests must be updated